packages/client/components/ReflectionGroup/ReflectionGroup.tsx

```diff
@@ -66,24 +66,37 @@ interface Props {
   reflectionGroup: ReflectionGroup_reflectionGroup
   swipeColumn?: SwipeColumn
   dataCy?: string
+  spotlightReflectionIds?: string[] | null
 }
 
 const ReflectionGroup = (props: Props) => {
-  const {meeting, openSpotlight, phaseRef, reflectionGroup, swipeColumn, dataCy} = props
+  const {
+    meeting,
+    openSpotlight,
+    phaseRef,
+    reflectionGroup,
+    swipeColumn,
+    dataCy,
+    spotlightReflectionIds
+  } = props
   const groupRef = useRef<HTMLDivElement>(null)
   const {localPhase, localStage} = meeting
   const {phaseType} = localPhase
   const {isComplete} = localStage
   const {reflections, id: reflectionGroupId, titleIsUserDefined} = reflectionGroup
+  const isSpotlightSource = spotlightReflectionIds?.length
```

Comment:
The name suggests it should be a boolean so maybe it's better to `!!spotlightReflectionIds?.length`